packages/modules-sdk/src/remote-query.ts

```diff
@@ -78,42 +80,43 @@ export class RemoteQuery {
   }
 
   public static getAllFieldsAndRelations(
-    data: any,
+    expand: RemoteExpandProperty | RemoteNestedExpands[number],
     prefix = "",
-    args: Record<string, unknown[]> = {}
+    args: JoinerArgument = {} as JoinerArgument
   ): {
     select: string[]
     relations: string[]
-    args: Record<string, unknown[]>
+    args: JoinerArgument
   } {
+    expand = JSON.parse(JSON.stringify(expand))
```

Comment:
Just curious, why not this? 

```suggestion
    expand = { ...expand }
```

Reply:
Because it is a shallow copy

Reply:
wut! TIL 😄